babel/zh.ini

```diff
@@ -0,0 +1,216 @@
+	批量刪除來自頻道的消息，n 指定返回查看的消息數量，'strict' 刪除所有消息，而不僅僅是與此機器人之間的消息。
+	如果您改為提供由破折號分隔的兩個消息 ID，clean 將在此範圍內運行，而不是從當前消息向上掃描。
+command_die_help = {p:local}{cmd} [saveconfig]
+	安全關閉機器人，如果您提供值，則保存配置文件
+janitor_set_success = 成功添加或更新此頻道的管理員。
+janitor_unset_success = 成功移除了該頻道的管理員。
+clean_success = 刪除 {n} 訊息成功。
+die_success = 正在關閉......
+
+[greeter]
+command_welcome_help = {p:local}{cmd} (get|set|clear)
+	控制服務器的歡迎消息
+	使用 `set` 獲取有關如何設置新歡迎消息的說明
+command_farewell_help = {p:local}{cmd} (get|set|clear)
+	控制服務器的告別信息
+	使用 `set` 獲取有關如何設置新告別消息的說明
+greeting_preview = in {channel}: {message}
+welcome_set_instructions = 要設置歡迎消息，使用 `{p:local}welcome set 歡迎， {} 到 {} 伺服器！` （舉例）
+	*第一個 {} 將始終成為用戶的提及，第二個 {} 將始終是當前服務器名稱。*
+welcome_set_success = 成功設置歡迎信息！
+welcome_clear_success = 刪除並禁用了歡迎消息。
+welcome_clear_failed = 您當前沒有設置歡迎消息。
+farewell_set_instructions = 要設置告別消息，使用 `{p:local}farewell set 再見， {} 來自 {} 伺服器！` （舉例）
+	*第一個 {} 將始終成為離開用戶的用戶名，第二個 {} 將始終是當前服務器名稱。*
+farewell_set_success = 成功設置告別信息！
+farewell_clear_success = 刪除並禁用了告別信息。
+farewell_clear_failure = 您當前沒有設置告別消息。
+
+[reactroles]
+command_reactrole_help = {p:local}{cmd} (prompt)
+	使用給定的反應提示創建一條消息。 您提供的每個反應都可以與角色相關聯。
+	角色將分配給任何對給定反應做出反應的用戶。
+setup1 = 對提示做出反應以添加反作用角色。 {canstop? *don't react to finish.*|}
+setup2 = 提及角色以將其添加到 {emoji}
+setup2_repeat = **這個表情符號已經被賦予了角色。**
+setup_cancel = 取消反應角色設置 *（每一步後超時為 30 秒）*
+setup_success = 反應角色設置完成！ 隨時刪除提示以撤消。
+role_granted = 給你這個角色 {roles} 。
+role_taken = 移除這個角色 {roles} 。
+
+[poll]
+command_poll_help = {p:local}{cmd} (title)
+	創建民意調查，您將被要求提供答案以及在提供標題後民意調查何時到期
+poll_preview = 投票預覽：
+setup1 = *回復加一個答案。*（隨時回复 '[stop]' 停止添加問題）
+setup_cancelled = 投票取消。
+setup2 = *回複分鐘數以設置倒計時。*
+setup2_failed = 使用默認倒計時
+poll_created = {author} 已開始投票。
+timenames = year,month,week,day,hour,minute,second
+timeplurals = years,months,weeks,days,hours,minutes,seconds
+prepositions = happened,happening closed,closing
+list_conjunction = ,
+list_last_conjunction = and
+far_past = 剩餘 {timelist} 關閉。
+near_past = 最近關閉。
+present = 現在關閉。
+near_future = 即將關閉。
+far_future = 在 {timelist} 結束。
+vote_cta = 對投票作出反應 | 這個民意調查是{multichoice?| not} 多項選擇
+no_winner = 投票 *"{title}"* 已過期，沒有投票。
+one_winner = *"{title}"* 投票的獲勝者是 **"{winner}"**
+multiple_winners = *"{title}"* 投票的 {num} 位獲勝者是 **"{winners}"**
+
+[dice]
+command_dice_help = {p:local}{cmd} [sides] [sides2] [sidesn] ...
+	最多可擲 8 個骰子，骰子的邊數為您指定的任意數量。 如果提供的數字無效，則跳過骰子。
+	默認為一個 6 面骰子
+roll_result = dice #{i}: {r}
+
+[emoji]
+command_emoji_help = {p:local}{cmd} (emoji name)
+	用機器人可以看到的帶有匹配名稱的表情符號回复
+	邀請 {c:main/botname} 到更多服務器以獲得更多表情符號
+command_emojicmd_help = {p:local}{cmd}
+	回復與提供的命令名稱一致的隨機表情符號
+not_found = 未能找到具有該名稱的表情符號。
+
+[meme]
+moving_news = **MemeDB 正在轉向自己的機器人（即將推出）**
+	稍後運行此命令以獲取添加新機器人的鏈接！
+
+[lightbulb]
+command_lightbulb_help = {p:local}{cmd} (enable|disable)
+	在您的服務器上啟用或禁用實驗性燈泡擴展。
+	每當燈泡看到可能可以提供幫助的消息時，它就會用燈泡表情符號做出反應。 做出反應，看看它能做什麼。
+enable_success = 燈泡現在已在此服務器上啟用，請嘗試發送類似 *"start a poll titled 'game tonight?'"* 或 *"roll a 10 sided dice"*
```

Comment:
This is correct, the text in quotes shouldn't be translated because lightbulb is only designed for English

Reply:
Please check the latest zh.ini is correct.
I mean, the command should be kept in English, if I translate it into Chinese, please let me know.